### PR TITLE
Revert "Cross-pool live migration: move CPU check to the target host"

### PR DESCRIPTION
**Pull request by lindig**
We see some errors in the context of snapshots that we need to investigate.

### CODE REVIEW

**Review by edwintorok — APPROVED** [review]
If we don't find another solution that fixes this reliably we can merge this revert (or perhaps we can merge this revert now, and once we found a fix and tested all the remaining corner cases then bring it back?)

**Comment by psafont** [comment]
See the fix in https://github.com/xapi-project/xen-api/pull/5148

**Review by robhoes — APPROVED** [review]

**Comment by psafont** [comment]
Needs a `git rebase master --signoff`

**Review by edwintorok — APPROVED** [review]

**Pull request merged**
